schemas/IDLs/org.gel.models.report.avro/6.0.0/CommonInterpreted.avdl

```diff
@@ -88,53 +92,496 @@ protocol CommonInterpreted {
+
+* `stand_alone`: `A`, stand-alone applied for benign variant critieria `(BA1)`
+* `supporting`: `P`, supporting applied for benign variant critieria `(BP1-6)` and pathogenic variant criteria `(PP1-5)`
+* `moderate`: `M`, moderate applied for pathogenic variant critieria (PM1-6)
+* `strong`: `S`, strong applied for pathogenic variant critieria (PS1-4)
+* `very_strong`: `S`, Very Stong applied for pathogenic variant critieria (PVS1)
+    */
+    enum AcmgEvidenceStrength{stand_alone, supporting, moderate, strong, very_strong}
+
+    /**
+    Each ACMG criterion is classified in one of these categories
+    */
+    enum AcmgEvidenceCategory{population_data, computational_and_predictive_data, functional_data, seggregation_data,
+        de_novo_data, allelic_data, other_database, other_data}
+    /**
+    Based on the category and the weight (A-VS-1-7) a criterion is defined this should identified each one of the
+    evidencies present in the guidelines
+    */
+    enum AcmgCriterion{BA1, BS1, BS2, BS3, BS4, BP1, BP2, BP3, BP4, BP5, BP6, BP7, PVS1, PS1, PS2, PS3, PS4, PM1, PM2,
+        PM3, PM4, PM5, PM6, PP1, PP2, PP3, PP4, PP5}
+    /**
+    Type of evidence in tge AMP guideline
+    */
+    enum AmpEvidenceType{mutation_type, therapies, variant_frequencies, potential_germline,
+        population_database_presence, germline_database_presence, somatic_database_presence, impact_predictive_software,
+        pathway_involvement, publications}
+    /**
+    AMP tier:
+* `TierI`: Variants of Strong Clinical Significance
+* `TierII`: Variants of Potential Clinical Significance
+* `TierIII`: Variants of Unknown Clinical Significance
+* `TierIV`: Benign or Likely Benign Variants
+    */
+    enum AmpTier{tierI, tierII, tierIII, tierIV}
+
+    /**
+    Categories of Clinical and/or Experimental Evidence as defined in AMP guidelines
+    */
+    enum AmpClincialOrExperimentalEvidenceCategory {therapeutic, diagnosis, prognosis}
+
+    /**
+    Levels for categories of Clinical and/or Experimental Evidence as defined in AMP guidelines
+    */
+    enum AmpClincialOrExperimentalEvidenceLevel{levelA, levelB, levelC, levelD}
+
+    /**
+    AcmgEvidence. This should be buit for each one of the evidences assing to a variants following the ACMG guidelines.
+    An AcmgEvidence, should map with one of the criteria defined, i.e, PVS1, BA1, PM1...
+    */
+    record AcmgEvidence {
         /**
-        Participant identifier
+        Evidence category as defined in ACMG guidelines
         */
-        string participantId;
+        union {null, AcmgEvidenceCategory} category;
+        /**
+        Evidence type: bening or pathogenic
+        */
+        union {null, AcmgEvidenceType} type;
+        /**
+        Weight categories as described in ACMG guideline
+        */
+        union {null, AcmgEvidenceStrength} strength;
+        /**
+        Intensity of the strength, together define the weight of each creteria, i.e the 2 in PM2
```

Comment:
should not reflect/indicate any level of intensity,  from the ACMG guidelines paper
"numbering within each category does not convey any differences of weight and is merely labeled to help refer to the different criteria."
I would rename it to something like "criteria" which seems to be how the ACMG guys name it and would keep the comment clarifying what it means.